tests/integration/lldp_test.py

```diff
@@ -122,7 +122,10 @@ def test_lldp_yaml(eth1_up):
         _send_lldp_packet()
         dstate = statelib.show_only((ETH1,))
         lldp_config = dstate[Interface.KEY][0][LLDP.CONFIG_SUBTREE]
-        test_neighbor = _get_lldp_test(lldp_config[LLDP.NEIGHBORS_SUBTREE])
+
+        assert len(lldp_config[LLDP.NEIGHBORS_SUBTREE]) == 1
+
+        test_neighbor = lldp_config[LLDP.NEIGHBORS_SUBTREE][0]
```

Comment:
Please, remove the introduced blank lines. Thanks!

Reply:
Done, thanks!